pkg/agent/nodeportlocal/k8s/annotations.go

```diff
@@ -80,18 +84,23 @@ func assignPodAnnotation(pod *corev1.Pod, nodeIP string, containerPort, nodePort
 		}}
 	}
 
-	current[NPLAnnotationStr] = toJSON(annotations)
+	current[NPLAnnotationKey] = toJSON(annotations)
 	pod.Annotations = current
 }
 
+func removePodAnnotation(pod *corev1.Pod) {
```

Comment:
Nit: I saw this func is called only once. Probably no need to add a func with two lines.

Reply:
The idea was to keep all annotation editing functions under `annotations.go` itself.

Reply:
Ok. Works for me.